app/jobs/socure_docv_results_job.rb

```diff
@@ -42,14 +42,64 @@ def perform(document_capture_session_uuid:, async: true, docv_transaction_token_
           passport_image_fingerprint: nil,
           selfie_image_fingerprint: nil,
         )
+
+        record_attempt(docv_result_response:, doc_pii_response:)
         return
       end
     end
+
+    record_attempt(docv_result_response:, doc_pii_response:)
     document_capture_session.store_result_from_response(docv_result_response)
   end
 
   private
 
+  def record_attempt(docv_result_response:, doc_pii_response: nil)
+    image_errors = {}
+
+    if socure_doc_escrow_enabled?
+      result = fetch_images(docv_result_response.to_h[:reference_id])
+      if result.is_a?(Idv::IdvImages)
+        images = result.attempts_file_data
+      else
+        image_errors = { image_request: [:network_error] }
+      end
+    else
+      images = {}
```

Comment:
this is technically not necessary, as `nil` [will splat as of ruby 3.4](https://github.com/ruby/ruby/pull/9477/files), but i think it's a little clearer/easier to reason about